### ISSUE TITLE
Can I get the original unicode of emoji?

### ISSUE THREAD

**Issue by BenYyyyyy** [headacheboy/data-of-multimodal-sarcasm-detection#17]
Hi,
We want to analysis the relationship between emoji and sarcasm. However, the form of the emoji code in your dataset is like emoji_XXX. I want to ask if I can know the mapping from the unicode to the emoji_XXX?

**Comment by HAPAVAN22** [comment]
Yes please @headacheboy can we get the original unicode mapping of the emojis?
